widget/embedded/src/pages/SettingsPage.tsx

```diff
@@ -1,93 +1,238 @@
+import { i18n } from '@lingui/core';
+import {
+  ChevronRightIcon,
+  Chip,
+  Divider,
+  InfoIcon,
+  List,
+  ListItemButton,
+  styled,
+  Switch,
+  TextField,
+  Typography,
+} from '@rango-dev/ui';
 import React from 'react';
-import { Settings } from '@rango-dev/ui';
-import { useSettingsStore } from '../store/settings';
-import { useMetaStore } from '../store/meta';
 import { useNavigate } from 'react-router-dom';
+
+import { Layout } from '../components/Layout';
 import { navigationRoutes } from '../constants/navigationRoutes';
-import { removeDuplicateFrom } from '../utils/common';
 import {
-  MAX_SLIPPAGE,
-  MIN_SLIPPGAE,
-  SLIPPAGES,
+  MAX_SLIPPAGE as maxSlippage,
```

Comment:
Please don't rename the import. Use the original name to communicate it's a app level constant.